src/components/NewLocationPage/LocationName/LocationName.style.tsx

```diff
@@ -19,3 +20,11 @@ export const MultiStateText = styled.div`
   font-size: 1rem;
   font-weight: 500;
 `;
+
+export const UpdatedOnText = styled.h2`
+  font-size: 1rem;
+  text-transform: uppercase;
+  line-height: 1;
+  margin: 0.25rem 0;
```

Comment:
i think we can drop the bottom margin (and maybe bump it up a tiny bit), so `margin-top: .5rem`